src/modules/interactivity.js

```diff
@@ -0,0 +1,27 @@
+// ---------- Declarations --------------------
+const toDoList = localStorage.getItem('toDoList') ? JSON.parse(localStorage.getItem('toDoList')) : [];
+const clearBtn = document.querySelector('.btn');
+
+// ---------- Functions -----------------------
+const updateStatus = (id) => {
+  const checkbox = document.querySelector(`.checkbox[data-id="${id}"]`);
+  const index = toDoList.findIndex((task) => task.index === +id);
+
+  toDoList[index].completed = checkbox.checked;
+  localStorage.setItem('toDoList', JSON.stringify(toDoList));
+  window.location.reload();
+};
+
+const clearCompleted = () => {
+  const filteredList = toDoList.filter((task) => task.completed !== true);
+  let i = 1;
+  filteredList.forEach((task) => {
+    task.index = i;
+    i += 1;
+  });
+
+  localStorage.setItem('toDoList', JSON.stringify(filteredList));
+  window.location.reload();
+};
+
```

Comment:
- [ ] While you did a wonderful job creating all the required functions to change the `completed` status :+1:, I have noticed that there are some issues related to Local Storage and adding information.
- [OPTIONAL] When you add a new task to your list the right button is not working, kindly fix that because it would help the users to get no confusion :+1: 
- When the user clicks on the check button right after adding some information it is not saved in the Local Storage, kindly fix the issue :+1: 
- Sometimes when you add more information to the list and click on the check button, that information is not saved. Kindly, check the function `updateStatus` to fix the issue :+1: 

Below you can see a video with all the issues that I mentioned before:


https://user-images.githubusercontent.com/105117832/221059420-0cf4482a-2376-452e-8f71-a4e904493aac.mp4